readthedocs/doc_builder/constants.py

```diff
@@ -1,9 +1,18 @@
 '''Doc build constants'''
 
+import os
+import re
+
 from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
 
 
+TEMPLATE_DIR = os.path.join(settings.SITE_ROOT, 'readthedocs', 'templates',
```

Comment:
I would say we want them in the doc_builder constants, to avoid cyclic imports for constants, but they should probably be prefixed to be sphinx specific.